app/controllers/api/memberships_controller.rb

```diff
@@ -1,5 +1,16 @@
 class API::MembershipsController < API::RestfulController
 
+  def add_to_subgroup
+    parent_group = current_user.groups.find_by_id!(params[:parent_group_id])
+    group = current_user.groups.find_by_id!(params[:group_id])
```

Comment:
I believe `load_and_authorize(:group)` does the same thing here, and will respond with the appropriate 403 instead of 500'ing if the user isn't a member of the specified group.